GVRf/Framework/framework/src/main/jni/objects/scene_object.cpp

```diff
@@ -445,31 +442,7 @@ int SceneObject::frustumCull(glm::vec3 camera_position, const float frustum[6][4
         return 1;
     }
 
-    // 3. Check if the object against the Level-of-details
-    // Transform the bounding sphere
-    glm::vec4 transformed_sphere_center(bounding_volume_.center(), 1.0f);
-
-    // Calculate distance from camera
-  //  glm::vec3 camera_position = camera->owner_object()->transform()->position();
-    glm::vec4 position(camera_position, 1.0f);
-    glm::vec4 difference = transformed_sphere_center - position;
-    float distance = glm::dot(difference, difference);
-
-    // this distance will be used when sorting transparent objects
-    render_data()->set_camera_distance(distance);
```

Comment:
Just realized that I should not have removed setting the camera distance. It is used by the state_sort.